M2/Macaulay2/packages/Macaulay2Doc/doc12.m2

```diff
@@ -629,37 +629,34 @@ document {
      Key => {ScriptedFunctor,
 	  (symbol^, ScriptedFunctor, Thing),
 	  (symbol_, ScriptedFunctor, Thing),
-	  (symbol SPACE, ScriptedFunctor, Thing)},
+	  Functor, (symbol SPACE, Functor, Thing)},
      Headline => "the class of all scripted functors",
      "A scripted functor accepts a subscript or a superscript:
      the primary example is ", TO "HH", ".",
-     SeeAlso => {"subscript", "superscript", "argument"}
+     Subnodes => TO \ {"subscript", "superscript", "argument"}
      }
 
 document {
      Key => argument,
-     Headline => "specify the function in a scripted functor for an argument",
-     TT "argument", " -- a key used in scripted functors under which is
+     Headline => "specify the function in a functor for an argument",
+     TT "argument", " -- a key used in functors under which is
      stored the function that accepts the arguments.",
-     SeeAlso => "ScriptedFunctor"
      }
 
 document {
      Key => subscript,
      Headline => "specify the function in a scripted functor for a subscript",
      TT "subscript", " -- a key used in scripted functors under which is
      stored the function of one variable that accepts the subscript and
-     returns a scripted functor that accepts the arguments.",
-     SeeAlso => "ScriptedFunctor"
+     returns a functor that accepts the arguments.",
      }
 
 document {
      Key => superscript,
      Headline => "specify the function in a scripted functor for a superscript",
      TT "superscript", " -- a key used in scripted functors under which is
      stored the function of one variable that accepts the superscript and
-     returns a scripted functor that accepts the arguments.",
-     SeeAlso => "ScriptedFunctor"
```

Comment:
If you remove this line, then the reader of this documentation page has no quick way to get to the documentation for scripted functors.

Reply:
Same as above.